content/tutorial-x-ray-image-processing.ipynb

```diff
@@ -0,0 +1,572 @@
+{
```

Comment:
<p>I think there's a typo here (2x): is it <strong>Sober</strong> filter or <strong>Sobel</strong> filter? On wikipedia, I see Sobel, but I may be missing something.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
Fixed 👍

Reply:
<p>There is still a Sober filter in the wikipedia link caption</p>

Reply:
Oh my! 🤦 Fixed it

Reply:
@melissawm thank you so much